numba/tests/test_array_manipulation.py

```diff
@@ -708,5 +712,30 @@ def _assert_raises(arr, val):
         _assert_raises(arr, val)
 
 
+    def test_shape(self):
+        pyfunc = numpy_shape
+        cfunc = jit(nopython=True)(pyfunc)
+
+        def check(x):
+            expected = pyfunc(x)
+            got = cfunc(x)
```

Comment:
Can we rename `got` to `received`? 'get', 'got' and 'gotten' are fairly unspecific.

Reply:
Whilst I appreciate that `got` and variants are not ideal, the continued use is consistent with the rest of the code base, therefore I'm inclined to leave it.

Reply:
Check, consistency wins.